admin/meetings/views.py

```diff
@@ -23,7 +23,7 @@ class MeetingListView(PermissionRequiredMixin, ListView):
     raise_exception = True
 
     def get_queryset(self):
-        return Conference.find()
+        return Conference.find().order_by('name')
```

Comment:
Driveby: `.find` is a modm style wrapper for django functions: https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/models/base.py#L98

Just use `Conference.objects.all().order_by('name')`